src/components/UserInfo.js

```diff
@@ -18,4 +18,12 @@ export default class UserInfo {
     this._aboutElement.textContent = about;
     this._avatarElement.src = avatar;
   }
+
+  updateAvatar(avatarUrl) {
```

Comment:
`updateAvatar` and `updateUserAndAvatar` are not needed in your project. Everything can be done with one `setUserInfo` because all 3 params comes from the server each time you edit the profile or avatar